build.gradle

```diff
@@ -87,9 +85,24 @@ javadoc {
 
 publishing {
     publications {
-        mavenJava(MavenPublication) {
+        maven(MavenPublication) {
             from components.java
+            groupId group
+            artifactId rootProject.name
+            artifact("$buildDir/artifacts/${project.name}" as File) {
+                classifier "jdk11"
+                extension "jar"
+                builtBy shadowJar
```

Comment:
I think we should add a different classifier for the shadow jar. Like `shadow` or `all`